rockme.txt

```diff
@@ -13,6 +13,7 @@ Blockchain
 Bring your own Device
 Buzzword
 Buzzword compliant
+clean Code
```

Comment:
Thanks for your contribution @FlorianJSa ! We have a convention regarding the buzzwords. Please write: `clean code` instead of `clean code`